src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -7252,428 +7267,422 @@ private void makeCoilRecipes() {
-                            GT_ModHandler.getModItem(Minecraft.ID, "stone", 64L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "grass", 64L, 0),
-                            GT_ModHandler.getModItem(TwilightForest.ID, "tile.TFSapling", 1L, 1),
-                            GT_Utility.getIntegratedCircuit(2) },
-                    Materials.Water.getFluid(10000L),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_TF", 1L, 0),
-                    300,
-                    30);
-            // Nether
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "netherrack", 64L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "soul_sand", 64L, 0),
-                            GT_ModHandler.getModItem(BiomesOPlenty.ID, "ash", 64L, 0),
-                            GT_Utility.getIntegratedCircuit(3) },
-                    Materials.Lava.getFluid(10000L),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_Ne", 1L, 0),
-                    300,
-                    30);
-            // End
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "end_stone", 64L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "obsidian", 64L, 0),
-                            GT_ModHandler.getModItem(Natura.ID, "Cloud", 64L, 1), GT_Utility.getIntegratedCircuit(4) },
-                    FluidRegistry.getFluidStack("ender", 10000),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_ED", 1L, 0),
-                    300,
-                    30);
-            // Far End Asteroids
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 0),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 1),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 2),
-                            GT_Utility.getIntegratedCircuit(5) },
-                    FluidRegistry.getFluidStack("endergoo", 10000),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_EA", 1L, 0),
-                    300,
-                    30);
+        // T0 Planets
+        // Overworld
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        getModItem(Botania.ID, "tinyPlanetBlock", 1, 0),
+                        getModItem(Minecraft.ID, "stone", 64, 0),
```

Comment:
(Optionally) Vanilla items can be referenced directly, without `getModItem` (`new ItemStack(Blocks.stone, 64)`).

Reply:
Is there a preferred way to do this for vanilla items? To me the `getModItem` method is more clear, but if the norm is to just use the ItemStack constructor directly then I can do that.

Reply:
It's OK to do it both ways. Difference only is that `getModItem` searches for item before creating `ItemStack`.
And as I said, it's optional.